fpm/recipes/terraform/recipe.rb

```diff
@@ -2,10 +2,10 @@ class Terraform < FPM::Cookery::Recipe
   name 'terraform'
   homepage 'https://www.terraform.io/'
 
-  version '0.11.1'
+  version '0.11.2'
 
   source "https://releases.hashicorp.com/terraform/#{version}/terraform_#{version}_linux_amd64.zip"
-  sha256 '4e3d5e4c6a267e31e9f95d4c1b00f5a7be5a319698f0370825b459cb786e2f35'
+  sha256 'f728fa73ff2a4c4235a28de4019802531758c7c090b6ca4c024d48063ab8537b'
```

Comment:
This `sha` isn't the same as the one I get when I download https://releases.hashicorp.com/terraform/0.11.2/terraform_0.11.2_linux_amd64.zip and run `shasum` against it. Am I doing something wrong?

Reply:
I was missing `-a 256`. It's fine!